stan/math/prim/mat/fun/scale_matrix_exp_multiply.hpp

```diff
@@ -0,0 +1,31 @@
+#ifndef STAN_MATH_PRIM_MAT_FUN_SCALE_MATRIX_EXP_MULTIPLY_HPP
+#define STAN_MATH_PRIM_MAT_FUN_SCALE_MATRIX_EXP_MULTIPLY_HPP
+
+#include <stan/math/rev/mat/fun/matrix_exp_action_handler.hpp>
```

Comment:
`prim` should not be bringing in `rev` files.  I don't think there is any use of `rev` within this include, though, so the solution is probably to move `stan/math/rev/mat/fun/matrix_exp_action_handler.hpp` to `stan/math/prim/mat/fun/matrix_exp_action_handler.hpp`

Reply:
Moved to `prim/mat/fun`